contracts/base/governance/ATMToken.sol

```diff
@@ -0,0 +1,160 @@
+     * @param amount The amount of tokens to mint
+     * @return true if successful
+     */
+    function mint(address account, uint256 amount) public onlyPauser() whenNotPaused() returns (bool) {
+        require(account != address(0), "ERC20: mint to the zero address");
+        _beforeTokenTransfer(address(0), account, amount);
+        _mint(account, amount);
+        return true;
+    }
+
+    /** @dev Creates `amount` tokens and assigns them to `account`, increasing
+     * the total supply.
+     *
+     * Includes a vesting period before address is allowed to use tokens
+     *
+     * Emits a {Transfer} event with `from` set to the zero address.
+     *
+     * Requirements
+     *
+     * - `to` cannot be the zero address.
+     *
+     * @param account The account which tokens will be assigned to
+     * @param amount The amount of tokens to be assigned
+     * @param vestingTime The length of the vesting period (in seconds)
+     */
+    function mintVesting(address account, uint256 amount, uint256 vestingTime) public onlyPauser() whenNotPaused() {
+        require(account != address(0), "ERC20: mint to the zero address");
+
+        _beforeTokenTransfer(address(0), account, amount);
+        // Should the totalsupply be updated here to avoid cap errors on successful vesting?
+        VestingTokens memory vestingTokens = VestingTokens(account, amount, block.timestamp+vestingTime);
+        _vestingBalances[account] = vestingTokens;
+        emit NewVesting(account, amount, vestingTime);
+    }
+
+    /**
+      * @notice Revokes the amount vested to an account
+      * @param account The account for which vesting is to be revoked
+      * @return true if successful
+      *
+     */
+     function revokeVesting(address account) public onlyPauser() whenNotPaused() returns (bool){
+         require(_vestingBalances[account].amount > 0, "Account does not have a vesting balance!");
+         VestingTokens memory vestingTokens = _vestingBalances[account];
+         uint256 revokedAmount = vestingTokens.amount;
+         uint256 revokedDeadline = vestingTokens.deadline;
+         delete _vestingBalances[account];
+         emit RevokeVesting(account, revokedAmount, revokedDeadline);
+         return true;
+     }
+
+    /**
+     *  @notice Checks if account has a vesting schedule
+     *  @param account The account being checked
+     *  @return true if successful
+     * 
+     */
+     function isVested(address account) public returns (bool) {
+         require(_vestingBalances[account].amount > 0, "Account does not have a vesting balance!");
+         return true;
+     }
+
+    /**
+     *  @notice Withdrawl of tokens upon completion of vesting period
+     *  @return true if successful
+     *
+     */
+     function withdrawVested() public returns (bool) {
+        VestingTokens storage vestingTokens = _vestingBalances[msg.sender];
+        require(vestingTokens.amount > 0, "Account does not have a vesting balance!");
```

Comment:
Please, change the error messages to be consistent with ours (using uppercase and underscore).

Example: ACCOUNT_DOESNT_HAVE_VESTING_BALANCE

Also, please verify the message length is less than 32 characters.

Reply:
Changed and verified